Code/autopkglib/URLDownloader.py

```diff
@@ -240,19 +240,15 @@ def main(self):
                     break
 
         retcode = proc.poll()
-        if retcode:
+        if retcode: # Non-zero exit coudse from curl => problem with download
```

Comment:
"Code". 

Sent from my iPhone

> On Jun 20, 2016, at 9:02 AM, Greg Neagle <notifications@github.com> wrote:
> 
> In Code/autopkglib/URLDownloader.py:
> 
> > @@ -240,19 +240,15 @@ def main(self):
> >                      break
> >  
> >          retcode = proc.poll()
> > -        if retcode:
> > +        if retcode: # Non-zero exit coudse from curl => problem with download
> What is the word "coudse" supposed to be?
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>